plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/language/CodeWhispererProgrammingLanguage.kt

```diff
@@ -26,6 +26,8 @@ abstract class CodeWhispererProgrammingLanguage {
 
     open fun isCodeScanSupported(): Boolean = false
 
+    open fun isFileScanSupported(): Boolean = false
```

Comment:
rename to a more descriptive one `AutoFileScan`?

Reply:
renamed to `isAutoFileScanSupported`